agent/command/git_merge_pr.go

```diff
@@ -19,17 +20,23 @@ import (
 	"github.com/pkg/errors"
 )
 
-type gitMergePr struct {
+type gitMergePR struct {
 	Token string `mapstructure:"token"`
 
 	statusSender send.Sender
 	base
 }
 
-func gitMergePrFactory() Command   { return &gitMergePr{} }
-func (c *gitMergePr) Name() string { return "git.merge_pr" }
+const (
+	mergePRAttempts      = 3
```

Comment:
should we use these settings since theyre defined already? 

https://github.com/evergreen-ci/evergreen/blob/2e726d4bdd0c0bede32f6e1f9ef9887f8392616a/thirdparty/github.go#L28